### ISSUE TITLE
App-header-backdrop-filter not for whole Ui

### ISSUE THREAD

**Issue by dhoeben** [home-assistant/iOS#2773]
**iOS device model, version and app version**

Model Name: iPhone 15 Pro
Software Version: iOS 17.5
App version: 2024.4

**Home Assistant Core Version**

2024.5.3

**Describe the bug**
When using css variable 'app-header-backdrop-filter' in theme, it applies only to the menu and not the top part of the iPhone screen. See screenshot.

**To Reproduce**
Add the `theme.yaml`:
`app-header-backdrop-filter: blur(20px)`
**Expected behavior**

**Screenshots**

![image](https://github.com/home-assistant/iOS/assets/61123717/73fb844b-ed6b-4c04-926b-eff9d23854dd)

**Additional context**

**Comment by bgoncal** [comment]
Indeed, I was discussing with the guys from frontend about this, it's not actually a bug but a nice improvement that I would also like to have it implemented.
To do that we need to extend the webview behind the iOS status bar and then use the safe area in CSS considerations for the header.
Currently I am working on high priority bugs but I'll keep you posted in case this gets implemented

**Comment by dhoeben** [comment]
Sounds good! Wish I could help, but unfortunately I am not good at all with Swift. Love your work btw!

**Comment by bgoncal** [comment]
I think it's better to make a complete implementation than to change the webview background to white on light and black on dark, because while it works for you, it may not look good for someone else, so.. in my opinion we shouldn't change this

**Comment by dhoeben** [comment]
It wouldn't make that much of a difference for anyone really, only for the people who are using opacity in their header background. 

If there's anything I can do, let me know.

**Comment by bgoncal** [comment]
Closing this here since we don't track enhancements in issues but I added a task to my backlog so it is implemented.
